### PR TITLE
Jenkins2 Deploy Box [Stage 2]

### PR DESCRIPTION
**Pull request by djsd123**
New node class to provision a Jenkins 2 instance

See: https://trello.com/c/RZCVbiHd

Requires: https://github.gds/gds/govuk-provisioning/pull/622

### CODE REVIEW

**Comment by afda16** [comment]
Jenkins build is failing, I think you need to add the new host to Hiera, in hosts::production::management::hosts:

**Comment by surminus** [comment]
Can we revert any changes that were previously made regarding upgrading `jenkins-1`?

File: hieradata/common.yaml [hunk 1]
```diff
@@ -1117,6 +1117,10 @@ hosts::production::management::hosts:
     ip: '10.1.0.3'
     legacy_aliases:
       - "deploy.%{hiera('app_domain')}"
+  deploy-1:
+    ip: '10.1.0.4'
+    legacy_aliases:
+      - "deploy.%{hiera('app_domain')}"
```
Comment (surminus) [inline hieradata/common.yaml]: I think this will conflict, can we change it to something different during testing?

**Comment by surminus** [comment]
I opened a PR to revert changes:

https://github.com/alphagov/govuk-puppet/pull/5644

This may affect what plugins you add, but we need to do this to get Puppet running on `jenkins-1` in Integration again (I have a new job I'd like to run on it).

**Pull request merged**